### PR TITLE
docs: sd-leadtext docs [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/styles/leadtext/leadtext.stories.ts [hunk 1]
```diff
@@ -24,14 +21,15 @@ export default {
...
+      'Large Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique. Facilisis commodo integer hendrerit tortor.'
+  }),
   argTypes
 };
 
-/**
- * Default: This shows sd-leadtext in its default state.
- */
-
 export const Default = {
```
Comment (coraliefeil) [inline packages/components/src/styles/leadtext/leadtext.stories.ts:33]: Remove default section https://solid-design-system.github.io/solid/feature_sd-leadtext-docs/?path=/docs/styles-sd-leadtext--docs#default pls

File: packages/components/src/styles/leadtext/leadtext.stories.ts [hunk 2]
```diff
@@ -41,22 +39,41 @@ export const Default = {
...
+        Large Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+        phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+        Facilisis commodo integer hendrerit tortor.
+      </div>
+    </div>
+  `
+};
+
 /**
  * Use the `inverted` class to make a leadtext with inverted colors.
```
Comment (coraliefeil) [inline packages/components/src/styles/leadtext/leadtext.stories.ts:66]: Use the ”inverted” class when displayed on primary background.

**Comment by coraliefeil** [comment]
[Figma is refined](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2706-32946&node-type=SECTION&t=23EIrSTNYEo83Lxr-11). Pls adress comments. THX

File: packages/components/src/styles/leadtext/leadtext.stories.ts [hunk 3]
```diff
@@ -42,21 +40,40 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a leadtext with inverted colors.
+ * Use `sd-leadtext` modifiers for alternative appearances:
+ * - Default: Extra large is the default leadtext size
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/leadtext/leadtext.stories.ts:44]:
```suggestion
 * - Extra Large is the default leadtext size
```

File: packages/components/src/styles/leadtext/leadtext.stories.ts [hunk 4]
```diff
@@ -42,21 +40,40 @@ export const Default = {
...
+ * - `sd-leadtext--size-lg`: Large can be used as an alternative to extra large.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/leadtext/leadtext.stories.ts:45]:
```suggestion
 * - `sd-leadtext--size-lg`: Large can be used as an alternative
```

File: packages/components/src/styles/leadtext/leadtext.stories.ts [hunk 5]
```diff
@@ -42,21 +40,40 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a leadtext with inverted colors.
+ * Use `sd-leadtext` modifiers for alternative appearances:
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/leadtext/leadtext.stories.ts:43]:
```suggestion
 * Use the `sd-leadtext` modifiers for alternative appearances:
```

Comment (auroraVasconcelos) [reply packages/components/src/styles/leadtext/leadtext.stories.ts:43]: Updated

File: packages/components/src/styles/leadtext/leadtext.stories.ts (hunk 4 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/leadtext/leadtext.stories.ts]: Updated

File: packages/components/src/styles/leadtext/leadtext.stories.ts (hunk 3 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/leadtext/leadtext.stories.ts]: Updated

**Review by MartaPintoTeixeira — APPROVED** [review]

**Pull request merged**